### DISCUSSION TITLE
Display: Darstellung der Daten verschiedener Invertern

### DISCUSSION THREAD

**Discussion by Maschler** [tbnobody/OpenDTU#781] [Q&A]
Hallo zusammen,

erstmal muss ich sagen das es ein richtig tolles Projekt ist und auf Anhieb alles funktioniert hat.

Bzgl. des Displays wollte ich fragen, wie und ob man auch die Daten von unterschiedlichen Invertern darstellten kann? Muss man da via Button weiterschalten oder etwas im Webinterface ändern? Ich sehe leider immer nur die Informationen von einem meiner beiden Inverter.

(Gleiches gilt für die Live-Daten)

Vielen Dank schon mal :)

**Comment by huckes** [comment]
Das Display zeigt die Summenwerte aller konfigurierten Wechselrichter an. Und im Web-Interface kannst Du dann zusätzlich die Werte der einzelnen Wechselrichter durchklicken.

**Reply by DPO99** [reply]
#955
